web-ng/src/app/components/main-page/main-page.component.ts

```diff
@@ -42,6 +43,7 @@ export class MainPageComponent implements OnInit {
   ) {
     // TODO: Make dynamic to support i18n.
     this.lang = 'en';
+    this.isSidePanelOpen = true;
```

Comment:
Might be better to name this `sideNavOpened` for consistency with component/attribute name and to avoid confusing name with a function.